### DISCUSSION TITLE
make prefab-gui-debug  error

### DISCUSSION THREAD

**Discussion by JackGod001** [efroemling/ballistica#315] [Q&A]
`Checking environment...
clang-format is required; please install it via apt, brew, etc.
make[1]: *** [.cache/checkenv] Error 1
make_prefab failed with error: Command '['make', 'prefab-mac-x86-64-gui-debug-build']' returned non-zero exit status 2.
make: *** [prefab-gui-debug-build] Error 255
`

**Comment by Dliwk** [comment]
Huh; this message tells you that you haven't `clang-format` program installed. I guess in mac you can fix this by installing `clang-format` via brew command.

**Reply by JackGod001** [reply]
I tried to install libogg libvorbis sdl2 python@3.8 cmake using conda environment but it doesn't seem to work. It seems that it can only be installed in the mac system environment? I tried to run brew install libogg libvorbis sdl2 python@3.8 cmake and it worked
